lib/JSCRunner.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+const fs = require('fs');
+const temp = require('temp');
+const inception = require('./inception');
+const ConsoleRunner = require('./ConsoleRunner');
+
+const errorRe = /^(?:.*?): (\w+)(?:: (.*))$/m;
+
+// JSC stack frame format:
+// StackFrames: StackFrame+
+// StackFrame: FunctionFrame | NativeFrame | EvalMarker
+// FunctionFrame: (FunctionName`@`)?SourceInfo
+// NativeFrame: (FunctionName`@`)?`[native code]`
+// FunctionName: .*
+// SourceInfo: File`:`Line`:`Column
+// File: .*
+// Line: \d+
+// Column: \d+
+// EvalMarker: `eval code`
+const frameRe = /(?:(.*)@)?(\[native code\]|(?:(.*):(\d+):(\d+)))/;
+
+const newGlobalTempFile = temp.path({ suffix: '.js' });
+fs.writeFileSync(newGlobalTempFile, 'arguments[0](this);');
```

Comment:
Ok, makes sense! (Also, this kind of stuff is why I think this library is so awesome :-P).
`runtimes` seems fine for this file. Want to make the change and I can pull this in?